### ISSUE TITLE
Delete tasks

### ISSUE THREAD

**Issue by rosymond** [CitizenScienceCenter/c3s-lab-client#46]
i just deleted 500 tasks for a draft project.
I clicked and nothing happened, no response of the system. So I clicked again ...
Only after about 1min I got the "Task successfully deleted" message(s).
Two issues:
- is it possible to optimize the delete process to be faster?
- in any case, it would be nice to have a spinning wheel (or simialr) to know the system is working
